code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -192,9 +197,7 @@
 
 /datum/ritual/cruciform/priest/heal_heathen/proc/heal_other(mob/living/carbon/human/participant)
 		to_chat(participant, "<span class='info'>A sensation of relief bathes you, washing away your some of your pain.</span>")
-		participant.reagents.add_reagent("laudanum", 5)
-		participant.adjustBruteLoss(-15)
-		participant.adjustFireLoss(-15)
+		participant.reagents.add_reagent("laudanum", 5, "meralyne", 5, "kelotane", 5)
```

Comment:
wew are you sure you want meralyne?

Reply:
While Meralyne does act fast as fuck to replicate the instant damage adjustment, this is a straight super buff. This will heal over time instead of instant, yes, but it can bring a body in crit to healthy like it was nothing.

Reply:
Will change later today to bicard.

Reply:
Also - reason chemsa re needed on this proc specifically is it does not compile when checking IS_SYNTH as it, for some reason, is undefined on this proc but not the others. So - workaround.

Reply:
I understand the reason, I'm stating the fact that meralyne is overkill and way too good, Bicard is decent enough